routes/articles.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const router = express.Router();
+
+const {
+  getArticlesBySource
+} = require('../controllers/articles.controller');
+
+router.get('/articles/:source', getArticlesBySource, (req, res) => {
+  res.render('articles', { articles: res.locals.articles });
+});
+
+//router.get('/articles/:source/:id', (req, res) => {
```

Comment:
As a practice our clients (and us) never allow commented out code to be merged to `main`.